lib/news/widgets/article_preview.dart

```diff
@@ -51,9 +51,9 @@ class ArticlePreview extends StatelessWidget {
               article?.title,
               style: theme.textTheme.display2,
             ),
-            FancyCachedBuilder<User>(
-              controller: article.authorId.controller,
-              builder: (_, author, __) {
+            EntityBuilder<User>(
+              id: article.authorId,
+              builder: handleEdgeCases((context, author, __) {
```

Comment:
We already handle loading manually.
```suggestion
              builder: handleError((context, author, __) {
```